add dobiss.markdown

kobedemetser opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.

Add dobiss.markdown
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [] Spelling, grammar or other readability improvements (`current` branch).
- [] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: [https://github.com/home-assistant/core/pull/140513]
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

## Summary by CodeRabbit

- **New Features**
  - Introduced comprehensive documentation for the Dobiss Home Assistant Integration.
  - Added a new section for upcoming patch releases, detailing specific dates and improvements for Home Assistant version 2025.3.
  - Enhanced documentation for various integrations, including AirGradient, Vodafone Station, and Comelit, outlining supported devices and functionalities.
  - Added a workaround for using Sonos devices with Spotify through the Music Assistant service.
  - Implemented a more structured filtering system for integrations on the website, improving user interface and experience.

- **Bug Fixes**
  - Corrected unit measurement issues and improved integration stability for SmartThings in upcoming patch releases.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

alvinchen1 committed:
Adding note about Climate entity

A helpful troubleshooting tip to resolve issues with BAF climate entity and some additional sensors.

See https://github.com/home-assistant/core/issues/130402

alvinchen1 committed:
Adjusted after reading HA doc standards url

alvinchen1 committed:
Removed double column nitpick from coderabbitai

alvinchen1 committed:
Additional nitpicks by coderabbitai

kylehakala committed:
Add terms and concepts, remove redundant phrasing, simplify intro

kylehakala committed:
Merge branch 'current' into zha-introduction

kylehakala committed:
Merge branch 'current' into zha-introduction

kylehakala committed:
Merge branch 'current' into zha-introduction

kylehakala committed:
Merge branch 'current' into zha-introduction

kylehakala committed:
Merge branch 'current' into zha-introduction

TurboJens committed:
Added windows instructions

I have added the information from https://github.com/home-assistant/home-assistant.io/issues/33676 and put in some extra details to make life easier for others.

TurboJens committed:
Update private_ble_device.markdown

TurboJens committed:
Updated formatting

TurboJens committed:
Merge branch 'current' into patch-1

c0ffeeca7 committed:
Features: add steps on customizing features on a card (#37833)

c0ffeeca7 committed:
Add card: add steps on customizing features, remove steps from tile card page (#37836)

c0ffeeca7 committed:
Create sections view: add substeps (#37837)

abmantis committed:
Minor fix to SmartThings sentence on blog post (#37843)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11899 to 0.5.11906 (#37846)

badlotus committed:
Update windows.markdown (#37825)

* Update windows.markdown

I noticed that there is a typo in the Windows installation instructions. Specifically, Step 3 of "Start Up Your Virtual Machine" ends with "(replace X.X.X.X with your ’s IP address).", when it should read "(replace X.X.X.X with your Windows server’s IP address)." 

Added {% assign board = "Windows server" %} to fix this typo that was generated on source/_includes/installation/operating_system.md.

* Start up VM: replace board variable by 'virtual machine' as VM is not a board

---------

Co-authored-by: c0ffeeca7 <k0ffeeka7@gmail.com>

jbouwh committed:
Add important note on MQTT broker settings when using Supla Cloud (#37839)

* Add important not on MQTT broker settings when using Supla Cloud

* typo

jaaviercr committed:
Update header-footer.markdown: fixed link to dashboard/statistic (#37848)

Removed the s from statisticS

bendikrb committed:
Removes no-NO from the language picker + a-z sorting (#37844)

* Sort languages by locale code

* Removes 'no-NO'

* Sort languages by language name

* Remove no-NO again (after sorting)

frenck committed:
2025.3.1 (#37861)

jbouwh committed:
Document how MQTT reports entity state updates (#37856)

* Document how MQTT reports entity state updates

* Update source/_integrations/mqtt.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

* Update source/_integrations/mqtt.markdown

Co-authored-by: J. Nick Koston <nick@koston.org>

---------

Co-authored-by: J. Nick Koston <nick@koston.org>

catsmanac committed:
Fix broken links in enphase envoy docs (#37862)

jbouwh committed:
Improve referrals to the device class documentation for MQTT entity platforms (#37858)

Petro31 committed:
Correct state and trigger based titles and add note (#37865)

Petro31 committed:
Fix trigger example with hidden template (#37800)

* Fix trigger example with hidden template

* Remove auto-TOC thing

jpbede committed:
Add known WebDAV limitations (#37878)

Diegorro98 committed:
Add 900 RPM option to washer spin speed options (#37888)

Commit:
Update philips_js.markdown

I've used the information from:
https://github.com/eslavnov/pylips/blob/master/docs/Home.md

to update the endpoints for the api version check.

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11906 to 0.5.11911 (#37894)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11906 to 0.5.11911.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
tiny tweaks

Commit:
Update philips_js.markdown

Use a Proper Heading Instead of Emphasized Text
Instead of using bold text for the section title

Commit:
Update philips_js.markdown

Made it generic, removed the Android, non-Android referneces

Commit:
Update philips_js.markdown

fixed misspelling, removed trailing white spaces

LG-ThinQ-Integration committed:
Update climate automation guide to LG ThinQ. (#37803)

* Update climate automation

* Fix textlint

* Update code block

---------

Co-authored-by: yunseon.park <yunseon.park@lge.com>

kiddyfurby committed:
Clarify requirement for Yale Conexis L1 (#37696)

There's a built-in bluetooth but that wouldn't work.
A extra module is required.

gwww committed:
Update environment_canada.markdown (#37874)

Tidy up language.

BastiaanBergman committed:
Added limitation to the documentation of python scripts. (#37889)

* Update python_script.markdown

Added limitation.

* Tweak

---------

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

c0ffeeca7 committed:
Map card: update screenshot to reflect current SW (#37895)

* Map card: update screenshot to reflect current SW

* Update map card to show cluster

mrdarrengriffin committed:
Uppercase distributor names to fix sorting

mrdarrengriffin committed:
Undo naming change

mrdarrengriffin committed:
Add alternative filter for case insensitive sort

mrdarrengriffin committed:
Lowercase mediarath

mrdarrengriffin committed:
Uppercase distributor names to fix sorting (#37902)

c0ffeeca7 committed:
OTBR: remove mention of mutliprotocol as it is discontinued (#37900)

mrdarrengriffin committed:
Fix sort order and filter placement (#37904)

* Move filter inside asset filter

* Change sort order

c0ffeeca7 committed:
Thread: remove mention of multiprotocol as it's been discontinued (#37901)

* Thread: remove mention of multiprotocol as it's been discontinued

* Update source/_integrations/thread.markdown

googanhiem committed:
Matter certification blog (#37907)

* Upload Matter certification blog

* Changes to image and some text updates

balloob committed:
Update title of markdown blog

dependabot[bot] committed:
build(deps): bump rack from 3.1.11 to 3.1.12

Bumps [rack](https://github.com/rack/rack) from 3.1.11 to 3.1.12.
- [Release notes](https://github.com/rack/rack/releases)
- [Changelog](https://github.com/rack/rack/blob/main/CHANGELOG.md)
- [Commits](https://github.com/rack/rack/compare/v3.1.11...v3.1.12)

---
updated-dependencies:
- dependency-name: rack
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

c0ffeeca7 committed:
build(deps): bump rack from 3.1.11 to 3.1.12 (#37910)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11911 to 0.5.11915

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11911 to 0.5.11915.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

c0ffeeca7 committed:
build(deps-dev): bump sorbet-runtime from 0.5.11911 to 0.5.11915 (#37913)

c0ffeeca7 committed:
convert to prereq section

c0ffeeca7 committed:
Apply suggestions from code review

c0ffeeca7 committed:
Update source/_integrations/philips_js.markdown

c0ffeeca7 committed:
fix indents

Co-authored-by: Joakim Plate <elupus@ecce.se>

c0ffeeca7 committed:
typo

c0ffeeca7 committed:
implement review feedback

c0ffeeca7 committed:
Update philips_js.markdown (#37891)

frenck committed:
2025.3.2 (#37917)

kkalitowski committed:
Update utility_meter.markdown

Grammar

jrhillery committed:
Update nexia.markdown to document emergency heat switch (#37918)

* Update nexia.markdown

Document emergency heat entities.

* Remove binary sensor documentation for emergency heat

Per review comment

* Say we add emergency heat if the device supports it

allenporter committed:
Update roborock documentation

allenporter committed:
Small fixes

allenporter committed:
Fix lint errors in bullet numbering

dependabot[bot] committed:
build(deps-dev): bump textlint from 14.4.2 to 14.5.0

Bumps [textlint](https://github.com/textlint/textlint) from 14.4.2 to 14.5.0.
- [Release notes](https://github.com/textlint/textlint/releases)
- [Commits](https://github.com/textlint/textlint/compare/v14.4.2...v14.5.0)

---
updated-dependencies:
- dependency-name: textlint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11915 to 0.5.11919

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11915 to 0.5.11919.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

c0ffeeca7 committed:
build(deps-dev): bump textlint from 14.4.2 to 14.5.0 (#37930)

c0ffeeca7 committed:
build(deps-dev): bump sorbet-runtime from 0.5.11915 to 0.5.11919 (#37931)

c0ffeeca7 committed:
tiny tweaks

c0ffeeca7 committed:
Update utility_meter.markdown (#37927)

joostlek committed:
Add more context to AirGradient introduction

allenporter committed:
Revert formatting for action arguments

allenporter committed:
Merge branch 'roborock-docs' of https://github.com/allenporter/home-assistant.io into roborock-docs

c0ffeeca7 committed:
Update roborock documentation to incorporate guidance in the quality scale (#37929)

dependabot[bot] committed:
build(deps-dev): bump json from 2.10.1 to 2.10.2 (#37946)

Bumps [json](https://github.com/ruby/json) from 2.10.1 to 2.10.2.
- [Release notes](https://github.com/ruby/json/releases)
- [Changelog](https://github.com/ruby/json/blob/master/CHANGES.md)
- [Commits](https://github.com/ruby/json/compare/v2.10.1...v2.10.2)

---
updated-dependencies:
- dependency-name: json
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

joostlek committed:
Add instructions to remove the AirGradient device (#37938)

joostlek committed:
Add installation parameters to AirGradient (#37939)

* Add installation parameters to AirGradient

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11919 to 0.5.11930

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11919 to 0.5.11930.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

c0ffeeca7 committed:
build(deps-dev): bump sorbet-runtime from 0.5.11919 to 0.5.11930 (#37949)

kobedemetser committed:
add dobiss.markdown

chemelli74 committed:
Add installation parameters to Comelit

chemelli74 committed:
Add installation parameters to Vodafone Station

chemelli74 committed:
add 2 more quality scale rules

chemelli74 committed:
typo

chemelli74 committed:
add 2 more quality scale rules

Nihvel committed:
add workaround to use spotify on sonos

c0ffeeca7 committed:
use configuration_basic block instead of configuration for UI config

c0ffeeca7 committed:
Move removal to end of doc as it's done at the end of the life cycle

c0ffeeca7 committed:
replace fetch by term poll because there a glossary entry for polling

c0ffeeca7 committed:
tiny tweaks

c0ffeeca7 committed:
Add glossary term

c0ffeeca7 committed:
tweak

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11930 to 0.5.11933

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11930 to 0.5.11933.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump rubocop from 1.73.2 to 1.74.0

Bumps [rubocop](https://github.com/rubocop/rubocop) from 1.73.2 to 1.74.0.
- [Release notes](https://github.com/rubocop/rubocop/releases)
- [Changelog](https://github.com/rubocop/rubocop/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rubocop/rubocop/compare/v1.73.2...v1.74.0)

---
updated-dependencies:
- dependency-name: rubocop
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump google-protobuf from 4.30.0 to 4.30.1

Bumps [google-protobuf](https://github.com/protocolbuffers/protobuf) from 4.30.0 to 4.30.1.
- [Release notes](https://github.com/protocolbuffers/protobuf/releases)
- [Changelog](https://github.com/protocolbuffers/protobuf/blob/main/protobuf_release.bzl)
- [Commits](https://github.com/protocolbuffers/protobuf/compare/v4.30.0...v4.30.1)

---
updated-dependencies:
- dependency-name: google-protobuf
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

c0ffeeca7 committed:
build(deps-dev): bump sorbet-runtime from 0.5.11930 to 0.5.11933 (#37965)

c0ffeeca7 committed:
build(deps-dev): bump rubocop from 1.73.2 to 1.74.0 (#37966)

c0ffeeca7 committed:
build(deps-dev): bump google-protobuf from 4.30.0 to 4.30.1 (#37967)

c0ffeeca7 commented:
Hi there, thank you for contributing, @kobedemetser :tada: 

A few things to get you started:

1. As you are adding a new integration, please rebase against the `next` branch, and target this PR against the `next` branch.
2. The link to the core PR should go to the "Link to parent pull request in the codebase:"
![image](https://github.com/user-attachments/assets/ca97a102-e525-46a5-8f60-2eb8d1e74b41)
3. Please add a link to the related PR in the brand repository, as indicated in the PR template.
4. Please fill out the rest of the PR template.
![image](https://github.com/user-attachments/assets/3ccf03bf-1e2b-48cd-ad7d-4223eac44278)

c0ffeeca7 commented:
I'm [drafting this PR](https://developers.home-assistant.io/docs/review-process#prs-are-being-drafted-when-changes-are-needed), as changes are needed. Please undraft it, once the changes have been implemented and the PR is ready for review again.

c0ffeeca7 committed:
tiny style tweaks

chemelli74 committed:
apply review comment

chemelli74 committed:
update link

chemelli74 committed:
Update source/_integrations/comelit.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Add 3 documentation rules to Vodafone Station (#37957)

chemelli74 committed:
lint

c0ffeeca7 committed:
Add 3 documentation rules to Comelit (#37956)

c0ffeeca7 committed:
Comelit: add subheadings

c0ffeeca7 committed:
Vodafone: add subheadings

c0ffeeca7 committed:
Comelit: add subheadings (#37969)

c0ffeeca7 committed:
Vodafone: add subheadings (#37970)

c0ffeeca7 committed:
Extract the ID from a BLE device using the PsExec tool from Microsoft (#37792)

Nihvel committed:
remove MA yaml example

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
add workaround to use spotify on sonos (#37963)

kobedemetser committed:
Add Dobiss integration documentation

frenck committed:
2025.3.3 (#37972)

c0ffeeca7 committed:
ZHA: Add terms and concepts, remove redundant phrasing, simplify intro (#37634)

c0ffeeca7 committed:
ZHA: convert terminology table into list

c0ffeeca7 committed:
Re-add titles

joostlek committed:
Update source/_integrations/airgradient.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Add more context to AirGradient introduction (#37937)

c0ffeeca7 committed:
ZHA: convert terminology table into list (#37979)

c0ffeeca7 committed:
Adding note to assist troubleshooting entities unavailable due to BAF fan losing connection to remote (#37075)

mrdarrengriffin committed:
Remove rel="noreferrer" from distributor links (#37980)

mrdarrengriffin committed:
Create .gitignore

mrdarrengriffin committed:
Add Pi Day recipes

mrdarrengriffin committed:
Remove old file

kobedemetser committed:
Merge branch 'home-assistant:current' into current

mrdarrengriffin committed:
Updated PDF

mrdarrengriffin committed:
Remove redundant file

mrdarrengriffin committed:
Pi day recipe (#37982)

kknopper committed:
Filters for integration page (#33644)

jbouwh committed:
Fix mqtt light schema defaults, state_value_template description and unclear template docs references

kylehakala committed:
Move zigbee 3.0 note to troubleshooting section

kylehakala committed:
format OTA section and collapse migrations

kylehakala committed:
Rearrange "adding devices" and "configuration" sections

kylehakala committed:
Move Advanced OTA update under OTA section

jbouwh committed:
Fix URL

jbouwh committed:
Correct description for payload_on and payload_off

tr4nt0r committed:
IPP integration has no quality scale

c0ffeeca7 committed:
IPP integration has no quality scale (#38021)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11933 to 0.5.11934

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11933 to 0.5.11934.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump nokogiri from 1.18.3 to 1.18.4

Bumps [nokogiri](https://github.com/sparklemotion/nokogiri) from 1.18.3 to 1.18.4.
- [Release notes](https://github.com/sparklemotion/nokogiri/releases)
- [Changelog](https://github.com/sparklemotion/nokogiri/blob/v1.18.4/CHANGELOG.md)
- [Commits](https://github.com/sparklemotion/nokogiri/compare/v1.18.3...v1.18.4)

---
updated-dependencies:
- dependency-name: nokogiri
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump rubocop-ast from 1.38.1 to 1.39.0

Bumps [rubocop-ast](https://github.com/rubocop/rubocop-ast) from 1.38.1 to 1.39.0.
- [Release notes](https://github.com/rubocop/rubocop-ast/releases)
- [Changelog](https://github.com/rubocop/rubocop-ast/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rubocop/rubocop-ast/compare/v1.38.1...v1.39.0)

---
updated-dependencies:
- dependency-name: rubocop-ast
  dependency-type: indirect
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

c0ffeeca7 committed:
ZHA: Rearrange sections (#38004)

c0ffeeca7 committed:
build(deps-dev): bump sorbet-runtime from 0.5.11933 to 0.5.11934 (#38023)

c0ffeeca7 committed:
build(deps): bump nokogiri from 1.18.3 to 1.18.4 (#38024)

c0ffeeca7 committed:
build(deps-dev): bump rubocop-ast from 1.38.1 to 1.39.0 (#38025)

dependabot[bot] committed:
build(deps): bump actions/setup-node from 4.2.0 to 4.3.0 (#38026)

c0ffeeca7 committed:
Fix mqtt light schema defaults, state_value_template description and unclear template docs references (#37999)

kylehakala committed:
Update source/_integrations/zha.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

kylehakala committed:
Merge branch 'current' into zha-refinement

c0ffeeca7 committed:
ZHA: general docs refinement (#38003)

mrdarrengriffin committed:
Add Domadoo for Yellow (#38031)

kylehakala committed:
Reformat limitations as pseudo-list and collapse it

dependabot[bot] committed:
build(deps-dev): bump sass-embedded from 1.85.1 to 1.86.0

Bumps [sass-embedded](https://github.com/sass-contrib/sass-embedded-host-ruby) from 1.85.1 to 1.86.0.
- [Commits](https://github.com/sass-contrib/sass-embedded-host-ruby/compare/v1.85.1...v1.86.0)

---
updated-dependencies:
- dependency-name: sass-embedded
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11934 to 0.5.11942

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11934 to 0.5.11942.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump rbs from 3.8.1 to 3.9.0

Bumps [rbs](https://github.com/ruby/rbs) from 3.8.1 to 3.9.0.
- [Release notes](https://github.com/ruby/rbs/releases)
- [Changelog](https://github.com/ruby/rbs/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ruby/rbs/compare/v3.8.1...v3.9.0)

---
updated-dependencies:
- dependency-name: rbs
  dependency-type: indirect
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

c0ffeeca7 committed:
build(deps-dev): bump sass-embedded from 1.85.1 to 1.86.0 (#38046)

c0ffeeca7 committed:
build(deps-dev): bump sorbet-runtime from 0.5.11934 to 0.5.11942 (#38047)

c0ffeeca7 committed:
build(deps-dev): bump rbs from 3.8.1 to 3.9.0 (#38048)

kylehakala committed:
Apply suggestions from code review

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
ZHA: Reformat several Troubleshooting sections and tips (#38043)

kobedemetser committed:
Merge branch 'home-assistant:current' into current

jbouwh commented:
Please open a new PR from a fresh branch from `next`
